### PR TITLE
Third try for the Mining Equipment Locker

### PR DESCRIPTION
**Pull request by ghost**
Because that would have been too easy if Duny's yearly Deff tweak didn't coincide with my PR being left to take dust

Old PR over at #5686, along with all the comments

The Mining Equipment Locker was a pretty huge snowflake. It was basically a vending machine in every single aspect, except the part where it was supposed to be coded as one

Now, it's just a vending machine, period. No reason to do otherwise
- Added Dwarven Mining Equipment, the vending machine that replaces Mining equipment lockers. It requires Mining access and vends everything the old machine used to vend, except for duckies and cigars
- The vending machine is linked to the Cargo account to emulate the old vending machine's behavior. Not that it will function correctly, given that the Cargo account is depreciated. This should only affect where the money is going (otherwise it's supposed to go to the station's account)
- The prices have been set way down. You can most likely deck out decently with your starting money. Some prize items like jetpacks and hookshots cost way more than your starting money, but then that's what Mining is for
- Expansion of items in the vending machine is possible, also maybe moving the machines on the outpost altogether
- All items from the old Mining Equipment Locker file was now moved to mine_items.dm. The old mining_equipment.dm itself now only contains the Ore Redemption Machine, and was renamed to this effect
- A recharge pack was added to use with the machine
- Maps updated to use the new vending machine, and Mining equipment's sprites were recycled whenever applicable

While I was in here, I used the access variables, otherwise nothing to signal, I guess

### CODE REVIEW

**Comment by PJB3005** [comment]
FUCKING LEARN TO REBASE.

This "I can't rebase because I'm dylan" is actually just so you can get rid of the comments.

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -2133,3 +2133,60 @@
...
 	pack = /obj/structure/vendomatpack/nuka
+
+/obj/machinery/vending/mining
+	name = "Dwarven Mining Equipment"
+	desc = "Get your mining equipment here, and above all keep digging!"
+	product_slogans = "This asteroid isn't going to dig itself!;Stay safe in the tunnels, bring two Kinetic Accelerators!;Jetpacks, anyone?"
+	product_ads = "Hungry, thirsty or unequipped? We have your fix!"
+	vend_reply = "What a glorious time to mine!"
+	icon_state = "mining"
+	req_access = list(access_mining)
```
Comment (PJB3005) [inline code/game/machinery/vending.dm]: No.

Comment (dylanstrategie) [inline code/game/machinery/vending.dm]: Very productive comment right here

I looked up req_access and even single items were in a list

Comment (PJB3005) [inline code/game/machinery/vending.dm]: *sigh I meant remove the fucking access requirement.

Comment (Shadowmech88) [inline code/game/machinery/vending.dm]: For what it's worth, I also disagree with making them access-restricted.

Comment (dylanstrategie) [inline code/game/machinery/vending.dm]: Well then, all access it is

**Comment by dylanstrategie** [comment]
And now ?

**Comment by PJB3005** [comment]
Oi nerd you said you moved the code, you didn't, you copy pasted it into a different file and unincluded the old one.

**Comment by PJB3005** [comment]
Alright now it's good.

**Comment by Probe1** [comment]
:-1: stop remaking the PR to avoid all the no's

**Comment by dylanstrategie** [comment]
If you have actual feedback to give, go ahead

**Comment by PJB3005** [comment]
It's fine now, except that the vendor needs recharging.

**Comment by dylanstrategie** [comment]
Normally, you just have to order a double pack at Cargo and slap the pack on the vendor

Hell, you can make brand new vending machines this way

**Comment by Probe1** [comment]
I did in the last two identical PRs.

**Comment by dylanstrategie** [comment]
I addressed all of it. If that's not the case, come again

File: code/datums/supplypacks.dm [hunk 2]
```diff
@@ -1350,6 +1350,15 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	containername = "Medical stack of packs"
 	group = "Vending Machine packs"
 
+/datum/supply_packs/miningmachines
+	name = "Mining stack of packs"
```
Comment (d3athrow) [inline code/datums/supplypacks.dm]: >mining stack of packs

Comment (dylanstrategie) [inline code/datums/supplypacks.dm]: I'll look to see if it can be named better

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -2133,3 +2133,59 @@
...
 	pack = /obj/structure/vendomatpack/nuka
+
+/obj/machinery/vending/mining
+	name = "Dwarven Mining Equipment"
+	desc = "Get your mining equipment here, and above all keep digging!"
+	product_slogans = "This asteroid isn't going to dig itself!;Stay safe in the tunnels, bring two Kinetic Accelerators!;Jetpacks, anyone?"
+	product_ads = "Hungry, thirsty or unequipped? We have your fix!"
+	vend_reply = "What a glorious time to mine!"
+	icon_state = "mining"
+	products = list(
```
Comment (DeityLink) [inline code/game/machinery/vending.dm]: at least keep the same order as the items in the mining locker

**Comment by dylanstrategie** [comment]
This really ought to be ready for merge now

**Comment by PJB3005** [comment]
inb4 dylan breaks everything

**Pull request merged**